unumpy/numpy_backend.py

```diff
@@ -24,11 +24,13 @@ def __ua_function__(method, args, kwargs):
     if method in _implementations:
         return _implementations[method](*args, **kwargs)
 
-    if not hasattr(np, method.__name__):
+    if hasattr(np, method.__name__):
```

Comment:
This should be replaced by checking the `__module__` of the `method`.

Reply:
Not sure about this :/

Reply:
You have to check `method.__module__`. It will be different depending on the submodule that the method is in.

For simplicity, you can also move everything in `_multimethods.py` to `__init__.py`.